srcpkgs/gcn64tools/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'gcn64tools'
+pkgname=gcn64tools
+_longname=raphnet-tech_adapter_manager
+version=2.1.19
+revision=1
+wrksrc="${_longname}-${version}"
+build_style=gnu-makefile
+make_use_env=true
+make_build_args="-C src"
+make_install_args="${make_build_args} PREFIX=\${DESTDIR}/usr"
```

Comment:
Isn't PREFIX already this?

Reply:
According to the manual, PREFIX is set to /usr, and make doesn't seem to install correctly except when PREFIX is under DESTDIR. This program's makefile treats PREFIX more or less as the absolute install directory.

Reply:
Sounds more like `make_use_env=false` so just remove that, and `make_install_args`.